NotificationApi/NotificationApi.AcceptanceTests/ApiTests/HearingConfirmation.feature

```diff
@@ -3,32 +3,38 @@ Feature: Create Hearing confirmation notifications
   As an API service
   I want to create hearing confirmation notifications data
 
-  Scenario: Create a hearing confirmation for a judge
-    Given I have a hearing confirmation for a judge email notification request
-    When I send the create notification request
-    Then the api client should return true
-    And Notify should have my request
+Scenario: Create a hearing confirmation for a judge
+	Given I have a hearing confirmation for a judge email notification request
+	When I send the create notification request
+	Then the api client should return true
+	And Notify should have my request
 
-  Scenario: Create a hearing confirmation for an ejud judge
-    Given I have a hearing confirmation for an ejud judge email notification request
-    When I send the create notification request
-    Then the api client should return true
-    And Notify should have my request
+Scenario: Create a hearing confirmation for an ejud judge
+	Given I have a hearing confirmation for an ejud judge email notification request
+	When I send the create notification request
+	Then the api client should return true
+	And Notify should have my request
 
-  Scenario: Create a hearing confirmation for a judicial office holder
-    Given I have a hearing confirmation for a joh email notification request
-    When I send the create notification request
-    Then the api client should return true
-    And Notify should have my request
+Scenario: Create a hearing confirmation for a judicial office holder
+	Given I have a hearing confirmation for a joh email notification request
+	When I send the create notification request
+	Then the api client should return true
+	And Notify should have my request
 
-  Scenario: Create a hearing confirmation for a LIP
-    Given I have a hearing confirmation for a LIP email notification request
-    When I send the create notification request
-    Then the api client should return true
-    And Notify should have my request
+Scenario: Create a hearing confirmation for a LIP
+	Given I have a hearing confirmation for a LIP email notification request
+	When I send the create notification request
+	Then the api client should return true
+	And Notify should have my request
 
-  Scenario: Create a hearing confirmation for a representative
```

Comment:
Should this test be removed?

Reply:
no because this is for normal participant